features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXAbortScenario.java

```diff
@@ -0,0 +1,30 @@
+package com.bugsnag.android.mazerunner.scenarios;
+
+import android.content.Context;
+
+import com.bugsnag.android.Configuration;
+
+import org.jetbrains.annotations.NotNull;
+
+public class CXXAbortScenario extends Scenario {
+
+    static {
+        System.loadLibrary("entrypoint");
+    }
```

Comment:
Likely not, every test case encapsulates all of the logic necessary to run the test case, and its harder to follow will increasingly deep inheritance trees. If there was shared logic beyond a few lines, it would be possible, but beyond the same 3-ish lines, the effort to ensure there's a shared and "generic enough" superclass is harder to deal with than reworking the individual cases.